src/Kdyby/Validator/DI/ValidatorExtension.php

```diff
@@ -82,6 +84,17 @@ public function loadConfiguration()
 			->setClass('Symfony\Component\Validator\Validator\ValidatorInterface')
 			->setFactory('Symfony\Component\Validator\Validator\RecursiveValidator');
 
+		Validators::assertField($config, 'strictEmail', 'boolean');
+
+		$builder->addDefinition($this->prefix('constraint.email'))
+			->setClass('Symfony\Component\Validator\Constraints\EmailValidator')
+			->setArguments(array(
```

Comment:
If there are arguments, somebody might have already registered it as a service to be able to set them, which might cause some collisions or hard to find bugs, depending on the order of the validators beeing registered.